app/gui/view/src/project/project_view_top_bar.rs

```diff
@@ -0,0 +1,69 @@
+//! Defines a UI container for the window control buttons and the "go to dashboard" button. This is
+//! merely here to make use of the auto-layout functionality.
+
+use ensogl::prelude::*;
+
+use enso_config::ARGS;
+use ensogl::application::Application;
+use ensogl::display;
+
+
+
+mod go_to_dashboard_button;
+pub mod window_control_buttons;
+
+
+
+// =================
+// === Constants ===
+// =================
+
+/// The gap in pixels between the various components of the project view top bar.
+const GAP: f32 = 18.0;
```

Comment:
I've written about it in the previous PR review - please take the values from the Figma design. The gap between components in the top bar is 16, not 18 pixels.

<img width="838" alt="image" src="https://github.com/enso-org/enso/assets/1623053/6809ce1a-47e0-45a5-9ce4-5275c382a6af">

Reply:
Ah, I understand the confusion. I took from here:
![x](https://github.com/enso-org/enso/assets/607786/ec9a5309-3382-42d0-b007-ac0f68f05d84)

But don't worry, I'll add the 2 pixels padding to the icon and change the gap to 16 pixels.

Reply:
That's measurement is not strictly correct. The right icon's width is 16 pixels, but its content is 12 pixels wide and it has 2 pixels of free space on both left and right side INSIDE of the icon. So the spacing between the icons is 16 px, and what you observe is just a part of the icon being transparent.